appveyor.yml

```diff
@@ -24,4 +24,4 @@ test_script:
   # Note that you must use the environment variable %PYTHON% to refer to
   # the interpreter you're using - Appveyor does not do anything special
   # to put the Python version you want to use on PATH.
-  - "%PYTHON%\\python.exe tests\\run_tests.py"
+  - "%PYTHON%\\python.exe -3 tests\\run_tests.py"
```

Comment:
is it worth commenting here to explain the `-3`. Does it assume a 2.7 interpreter (it's currently `python.exe`)

Reply:
Actually, I didn't intend to put -3 in the automated builds. The errors are the same on every platform. Having them in the local build is good enough.